pkg/gui/commits_panel.go

```diff
@@ -36,7 +36,7 @@ func (gui *Gui) branchCommitsRenderToMain() error {
 		task = types.NewRenderStringTask(gui.c.Tr.NoCommitsThisBranch)
 	} else {
 		cmdObj := gui.git.Commit.ShowCmdObj(commit.Sha, gui.State.Modes.Filtering.GetPath(),
-			gui.IgnoreWhitespaceInDiffView)
+			gui.IgnoreWhitespaceInDiffView, gui.UserConfig.Gui.DetectRenames)
```

Comment:
This is about the only thing I'm not 100% on (since my Go knowledge is not good at all).

If there's a better way to pull out the user config option from here, then please do let me know!